rasa_composite_entities/composite_entity_extractor.py

```diff
@@ -7,11 +7,21 @@
 from rasa.__main__ import create_argument_parser
 from rasa.data import get_core_nlu_files
 from rasa.nlu.extractors import EntityExtractor
-from rasa.nlu.training_data.loading import _guess_format
+import itertools
+import os
+import re
+import tempfile
+import warnings
+
+from rasa.__main__ import create_argument_parser
+from rasa.data import get_core_nlu_files
+from rasa.nlu.extractors import EntityExtractor
+from rasa.nlu.training_data.loading import _guess_format, guess_format
 from rasa.nlu.utils import write_json_to_file
 from rasa.utils.io import list_files, read_json_file
 
 COMPOSITE_ENTITIES_FILE_NAME = "composite_entities.json"
+COMPOSITE_PATTERNS_KEY = "composite_entity_patterns"
```

Comment:
The key should contain the word `path` in it. I would probably use `composite_patterns_path`.